paasta_tools/marathon_tools.py

```diff
@@ -265,6 +265,14 @@ def get_bounce_method(self):
         :returns: The bounce method specified in the config, or 'crossover' if not specified"""
         return self.config_dict.get('bounce_method', 'crossover')
 
+    def get_dockerfile_location(self):
+        """Get the URI for the .dockerfile.
+
+        :param service_config: The service instance's configuration dictionary.
+        :returns: The URI specified, or file:///root/.dockerfile if unspecified."""
+
+        return self.config_dict.get('dockerfile_location', 'file:///root/.dockercfg')
```

Comment:
Do we want to pull this path out into a constant somewhere like `DEFAULT_SOA_DIR`?